neet/synchronous.py

```diff
@@ -177,7 +177,54 @@ def basins(net):
 
     return nx.weakly_connected_component_subgraphs(g)
 
+def timeseries(net, timesteps, size=None):
```

Comment:
Note that you get a MethodError when trying to input a WTNetwork with a size!=None (since WTNetwork.state_space doesn't take any arguments since it can't be variable size). Might want to raise a more specific error for this.